Playbooks/playbook-Detonate_File_-_Lastline.yml

```diff
@@ -0,0 +1,438 @@
+id: Detonate File - Lastline
+version: -1
+name: Detonate File - Lastline
+fromversion: 4.0.0
+description: |-
+  Detonates a File using the Lastline sandbox.
+  Lastline supports the following File Types:
+
+  32-bit Portable Executables (PE)files; 64-bit PE+files
+  exe, sys, dll, com, scr, cpl, ocx, cgi
+  Microsoft Office Suite documents
+  doc,dotm, docx, dotx, xls, ppam, xlsx, pps, xlsb, ppsx, xlsm, ppsm, ppt, ppt, pptx, pptm, rtf, shs, xltm, sldm, xltx, sldx, xlam, thmx, docm, xar
+  Just Systems Ichitaro documents
+  jtd, jtdc
+  Adobe
+  pdf, swf
+  Compressed files
+  gz, 7z, tgz, msi, zip, lzh, cab, lzma, rar
+  Android application package
+  apk, Java, JAR, CLASS, Java Script, Java bin files
+  Image files
+  jpeg, png, gif
+  Other file types
+  cmd, ace, bat, arj, vbs, chm, xml, lnk, url, mof, htm, ocx, html, potm, eml, potx, msg, ps1, vb, reg, vba, wsc, vbe, wsf, vbs, wsh
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 489b704c-bef9-40e7-8aa8-f74cca308a94
+    type: start
+    task:
+      id: 489b704c-bef9-40e7-8aa8-f74cca308a94
+      version: -1
+      name: ""
+      description: '-'
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "7"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 50,
+          "y": 50
+        }
+      }
+    note: false
+  "1":
+    id: "1"
+    taskid: f142846a-82fb-4224-8c83-0debc6838bf1
+    type: regular
+    task:
+      id: f142846a-82fb-4224-8c83-0debc6838bf1
+      version: -1
+      name: Lastline Upload File
+      description: Uploads the submission to McAfee ATD.
```

Comment:
Please make sure descriptions are changed to reflect this integration.

Reply:
Fixed for all fields.